.gitlab-ci.yml

```diff
@@ -54,7 +53,7 @@ Push:
     - push
   script:
     - cd ${HOME}/go/src/github.com/${REPONAME}/${IMAGENAME}
-    - sudo make dockerpush
+    - sudo make dockerops
```

Comment:
Check the build and push step from circleci yaml

Reply:
Build the image using: https://github.com/litmuschaos/chaos-operator/blob/master/.circleci/config.yml#L50

and for push the image use the approach
- https://github.com/litmuschaos/chaos-operator/blob/master/.circleci/config.yml#L74
- https://github.com/litmuschaos/chaos-operator/blob/master/.circleci/config.yml#L90
-

Reply:
You can do this in next PR